.github/workflows/run-e2e.yml

```diff
@@ -57,7 +57,7 @@ jobs:
           publish_branch: gh-pages/dimpact-e2e-test-report # deploying to gh-pages branch
           keep_files: true # retains files that are not part of the current publish
       - name: Notify Slack
-        if: always()
+        if: ${{ github.ref == 'refs/heads/master' }}
```

Comment:
We don't use `master`. Nobody should. ;-) It is `main` we use.

Reply:
Also the `${{ }}` are not needed I think.

Reply:
Copy-past**a** error

Reply:
Fixed